service/awsconfig/v7/key/key.go

```diff
@@ -32,6 +32,11 @@ const (
 	PolicyNameTemplate = "EC2-K8S-Policy"
 )
 
+const (
+	WorkerASGKey = "WorkerASGName"
+	WorkerASGTag = "workerAutoScalingGroup"
```

Comment:
The Tag in the var name confuses me as this isn't an AWS resource tag. `WorkerASGName` is clearer for me.

Reply:
What about `Ref`? IMO it is not the name but as I saw it in AWS terms it is the tag for the ref in order to reference the name.

Reply:
Yes `Ref` is good. Thx!

Reply:
Cool. Done.